### ISSUE TITLE
Datepicker modal won't appear on IE 11 [v.1.0.0]

### ISSUE THREAD

**Issue by seeker5084** [Dogfalo/materialize#5483]
[Materialize 1.0.0-alpha.2]

IE 11 won't show up 'datepicker' modal.
I clicked input field, but nothing happened (even no errors or warnings!).

Other browsers are just fine.

Here is my code

- HTML5
```
...
<form method="post" action="">
    <div class="row">
        <div class="input-field col s12">
            <input type="text" class="datepicker validate" id="date_picker" name="date_picker" placeholder="Choose a date" />
            <label for="date_picker">Pick a date</label>
        </div>
    </div>
</form>
...
```

- JS (using jQuery 3.2.1)
```
$(document).ready(function(){
    $('.datepicker').datepicker();
});
```

[1.0 docs' example](http://next.materializecss.com/pickers.html) also doesn't work on ie 11...

Need help.

**Comment by robbiemilejczak** [comment]
This error is thrown at line 11208 of materialize.js

`this.wrapper = document.createElement('div')`;
`this.wrapper.classList.add()`;

On Chrome/FF this creates an element of the shape `<div class></div>` but on IE11 the error `Argument not optional` is thrown. I am assuming this is not intentional because I can't think of a reason to call `classList.add()` without arguments. Immediately after it is called, `$(this.wrapper).addClass('select-wrapper')` is called. If the line `this.wrapper.classList.add()` is removed the code seems to behave the same way. Ideally, this is an easy fix and I'm happy to make a PR

**Comment by Dogfalo** [comment]
Fixed in 7c4fc588f5ae3d4d3e838354ab5195905a2a4184
